TODO.txt

```diff
@@ -17,10 +16,14 @@ Version 0.14
   deprecation warning test for this alias.
 * Remove deprecated ``sigma_range`` kwargs in ``skimage.restoration.denoise_bilateral``
   and corresponding tests.
+* Remove deprecated `normalise` and corresponding test for ``skimage.feature.hog``. 
```

Comment:
Yes, we could discuss this. I've the following thinking: as we hadn't _exposed_ the deprecation earlier (no warnings, etc), if it more _fair_ to start it from the 0-point. I don't mind scheduling it to 0.13 (i.e. refactoring the code before the next release).
I would like to hear some opinions from the project owners.